student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] The Student should have a method that serves as the 'belongs-to' side (a student belongs to a classroom)
- [ ] It's a requirement that we set up the classroom for students and that WHEN setting the classroom for a student we also add it to the classrooms' students. Our `Student` class should look similar to the following

e.g
```
class Student < Person
  attr_reader :classroom
 
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```